src/root/Routes.tsx

```diff
@@ -445,6 +447,34 @@ class Routes extends React.Component<Props & RouteComponentProps, AppState> {
     this.rpc.clearTimers();
   };
 
+  shieldSaplingBalanceToOrchard = async () => {
+    try {
+    const result = await this.rpc.shieldSaplingBalanceToOrchard()
+
+    if (!result.includes('txid')) {
+      throw new Error(result)
```

Comment:
I think the user needs some feedback here, you can use `openerrormodal` here if the result have no `txid`
And the user needs some feedback even if the transaction finished successfully... using this `txid` as well.